common/kvstore/src/test/java/org/apache/spark/util/kvstore/CustomType1.java

```diff
@@ -52,12 +50,7 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return Objects.toStringHelper(this)
-      .add("key", key)
-      .add("id", id)
-      .add("name", name)
-      .add("num", num)
-      .toString();
+    return "CustomType1[key=" + key + ",id=" + id + ",name=" + name + ",num=" + num;
```

Comment:
(Can't use Apache Commons Lang3 here.)

Reply:
why not use String Interpolation, e.g. s"CustomType1[key=$key, id=$id, name=$name, num=$num]"?

Reply:
(This is Java)

Reply:
oh, sorry. Now I know why scala is better language. :)